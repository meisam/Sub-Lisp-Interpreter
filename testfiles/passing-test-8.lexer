LispOpenParentheses
(
LispLiteralAtom
DEFUN
LispLiteralAtom
NOTSOSILLY
LispOpenParentheses
(
LispLiteralAtom
A
LispLiteralAtom
B
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
COND
LispOpenParentheses
(
LispOpenParentheses
(
LispLiteralAtom
EQ
LispLiteralAtom
A
LispNumericAtom
0
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
PLUS
LispLiteralAtom
B
LispNumericAtom
1
LispCloseParentheses
)
LispCloseParentheses
)
LispOpenParentheses
(
LispOpenParentheses
(
LispLiteralAtom
EQ
LispLiteralAtom
B
LispNumericAtom
0
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
NOTSOSILLY
LispOpenParentheses
(
LispLiteralAtom
MINUS
LispLiteralAtom
A
LispNumericAtom
1
LispCloseParentheses
)
LispNumericAtom
1
LispCloseParentheses
)
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
T
LispOpenParentheses
(
LispLiteralAtom
NOTSOSILLY
LispOpenParentheses
(
LispLiteralAtom
MINUS
LispLiteralAtom
A
LispNumericAtom
1
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
NOTSOSILLY
LispLiteralAtom
A
LispOpenParentheses
(
LispLiteralAtom
MINUS
LispLiteralAtom
B
LispNumericAtom
1
LispCloseParentheses
)
LispCloseParentheses
)
LispCloseParentheses
)
LispCloseParentheses
)
LispCloseParentheses
)
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
NOTSOSILLY
LispNumericAtom
2
LispNumericAtom
4
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
NOTSOSILLY
LispNumericAtom
3
LispNumericAtom
5
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
NOTSOSILLY
LispNumericAtom
0
LispNumericAtom
0
LispCloseParentheses
)
