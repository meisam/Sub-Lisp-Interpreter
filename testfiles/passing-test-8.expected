NOTSOSILLY
11
253
1

